### DISCUSSION TITLE
where can I download the electribe firmware?

### DISCUSSION THREAD

**Discussion by MUNKE23** [bangcorrupt/hacktribe#247] [Q&A] [answered]
Download Electribe 2 Sampler firmware version 2.02 and move SYSTEM.VSB to hacktribe directory

**Comment by BKLronin** [comment, marked as answer]
https://cdn.korg.com/us/support/download/files/0b87bcd3112fbb8c0ad7b0f55e618837.zip?response-content-disposition=attachment%3Bfilename%2A%3DUTF-8%27%27electribe_sampler_system_v202.zip&response-content-type=application%2Foctet-stream%3B

**Comment by MUNKE23** [comment]
thanks!
